### DISCUSSION TITLE
How to copy cell output of Interactive window of VS code just with keyboard?

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [microsoft/vscode-jupyter#16146] [Questions and Answers]
Hello, as a newcomer in VS code, I adore it alot. 
As a hardcore vim user, who just joined VS code community, it is so much of production lost if I have to select and copy my errors with mouse. I just wanted to ask you if you can help me to find out any ways to select output with keyboards?

P.S. I could manage to change the focus to IW with CTRL+1 and CTRL+2 but can not select the outputs.
thanks 
![image](https://github.com/user-attachments/assets/238deb84-75e5-4ab8-be32-da8f19165e59)

**Comment by DonJayamanne** [comment]
@amunger /cc

**Comment by amunger** [comment]
answered here https://github.com/microsoft/vscode-jupyter/issues/15948#issuecomment-2438658272, and the linked issue from there
